package.json

```diff
@@ -11,6 +11,9 @@
 		"type": "git",
 		"url": "https://github.com/dojo/loader.git"
 	},
+	"dependencies": {
+		"dojo-core": "dojo/core"
```

Comment:
Oh, I see, he used it for the text plugin...  which is clearly going to involve XHR.  This is problematic and maybe something we didn't think about?